atomicapp/nulecule/container.py

```diff
@@ -122,13 +122,19 @@ def extract_files(self, image, source, dest):
                   '%s:/%s' % (container_id, source), dest]
         logger.debug(
             'Copying data from docker container: %s' % ' '.join(cp_cmd))
-        subprocess.check_output(cp_cmd)
+        try:
+            subprocess.check_output(cp_cmd)
+        except subprocess.CalledProcessError as e:
+            pass
```

Comment:
Errrrrr. I don't really understand what's going on here. If it errors out it will ignore it?

I was thinking that if it errors, it should log the info and fail `logger.error("foo bar")`